onnxruntime/core/providers/partitioning_utils.cc

```diff
@@ -156,7 +156,9 @@ std::vector<std::vector<const Node*>> CreateSupportedPartitionNodeGroups(
     const Node& node = *nodes_to_process.front();
     nodes_to_process.pop_front();
 
-    const bool is_node_supported = is_node_supported_fn(node);
+    const bool is_node_supported =
+        node.GetExecutionProviderType().empty() &&  // a node that is already assigned to an EP is unsupported
```

Comment:
In this case, maybe log some info out? I assume this should not happen

Reply:
an existing value from GetExecutionProviderType() is not unexpected, for example, if another EP has already taken the node